cvmfs/compat.h

```diff
@@ -647,10 +647,6 @@ class StringHeap : public SingleCopy {
  private:
   void AddBin(const uint64_t size) { assert(false); }
 
-  uint64_t size_;
```

Comment:
This is in fact not dead code.  It's necessary for the hot patch in order to establish the memory layout of that class in a previous version of cvmfs.

Reply:
All the file or just `size_` ?

Reply:
All of them

Reply:
Removed that file!